### PR TITLE
Faster HR graphics on the TS2068

### PR DESCRIPTION
**Pull request by zx70**
#1274

### CODE REVIEW

File: libsrc/target/ts2068/graphics_hr/w_clga_callee.asm [hunk 1]
```diff
@@ -1,47 +1,55 @@
 
+; "Clear area" on the TS2068
```
[the remark this answers is not in the document]
Comment (zx70) [reply libsrc/target/ts2068/graphics_hr/w_clga_callee.asm:2]: Got it.   A new commit will follow shortly.

[the remark this answers is not in the document]
Comment (zx70) [reply libsrc/target/ts2068/graphics_hr/w_clga_callee.asm:2]: I forgot that probably this common directory probably includes the SAM and that the standard Spectrum mode now coexists with the high resolution one on the TS2068, right?  My idea is to tune this single function and use it as a model for the rest.

[the remark this answers is not in the document]
Comment (suborb) [reply libsrc/target/ts2068/graphics_hr/w_clga_callee.asm:2]: Yes. There’s commonality between the 4 machines for MODE0

Commonality with zxn for all the timex modes. 

The lst files should make sense of it.

[the remark this answers is not in the document]
Comment (zx70) [reply libsrc/target/ts2068/graphics_hr/w_clga_callee.asm:2]: https://github.com/z88dk/z88dk/blob/4f0058793ca50a45861d999a1e148664246d40f6/libsrc/target/zx-common/graphics/w_clga_callee.asm#L42

I'm going to extend this logic, I hope the variable is always available

**Pull request merged**